### ISSUE TITLE
New launcher of the game

### ISSUE THREAD

**Issue by TREBee** [long-war-2/lwotc#1116]
I have a problem with launching the LWOTC since the new game launcher was released by Steam (after the release of Chimera Squad). The new launcher does not see the downloaded mods. previously the game was working fine with the mods. How can I solve this problem?

**Comment by wdanny69** [comment]
hello. first off you are using the steam launcher which won't recognize the mods unless its from the workshop. if you read the instructions, it has you use the alternative mod launcher which you can find by just looking that name and then you can follow the rest of the instructions listed in the wiki. the aml has a much better ui and helps manage mods way better anyways.

**Comment by charly1r** [comment]
1. Steam Workshop MODS showing up in the launcher but not loading
After I tried AML and wanted to go back to the Regular launcher all the mods from the "WORKSHOP" that had exclamation marks were not loading, they are marked because they specifically are not WOTC compatible.
(Steam has more controller config options for the regular launcher but I will still use AML for troubleshooting purposes)

I found an article that made them work again:
On the broken mod folder, find the "ModName.XcomMod" file
insert this, just after the description line:
RequiresXPACK=true

By doing this it removes the exclamation mark and loads the broken mods from the workshop, steam workshop path:
...\Steam\steamapps\workshop\content\268500

2. I have been testing and found out a way that I can use downloaded mods like lwotc and highlander by putting them in 2 different folder locations:
...\Steam\steamapps\common\XCOM 2\XComGame\Mods
...\Steam\steamapps\common\XCOM 2\XCom2-WarOfTheChosen\XComGame\Mods

I am no expert but I think this location make them appear in the launcher ...\Steam\steamapps\common\XCOM 2\XComGame\Mods
And the other location is where the actual mod runs from ...\Steam\steamapps\common\XCOM 2\XCom2-WarOfTheChosen\XComGame\Mods
